### PR TITLE
Added KG60 and KG65 to QMK

### PR DESCRIPTION
**Pull request by mrnoisytiger**
## Description

Added KG60 and KG65 to QMK. Cloned from FJLabs TF60 and TF65 and changed VID/PID as well as all FJLabs references to reflect Keygem due to branding.

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/keygem/kg60ansi/readme.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
+# Keygem KG60 ANSI 
+
+The following is the QMK Firmware for the Keygem KG60 ANSI PCB, a universal tray mount 60% hotswap PCB with per key RGB LED's. 
+
+The PCB will feature:
+* Kailh Hotswap sockets
+* QMK & VIA compatibility
+* Per-Key RGB LED's
+
+---
```
Comment (drashna) [inline keyboards/keygem/kg60ansi/readme.md:10]:
```suggestion
```

Comment (drashna) [inline keyboards/keygem/kg60ansi/readme.md:10]:
```suggestion
```

File: keyboards/keygem/kg65rgbv2/readme.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
+# Keygem KG65 RGB V2 
+
+The following is the QMK Firmware for the FJLabs TF65 RGB V2 PCB, a universal tray mount 65% hotswap PCB with per key RGB LED's. 
+
+The PCB will feature:
+* Kailh Hotswap sockets
+* QMK & VIA compatibility
+* Per-Key RGB LED's
+
+---
```
Comment (drashna) [inline keyboards/keygem/kg65rgbv2/readme.md:10]:
```suggestion
```

**Review by drashna — CHANGES_REQUESTED** [review]

**Review by drashna — APPROVED** [review]

**Comment by mrnoisytiger** [comment]
Bump. 😊

File: keyboards/keygem/kg60ansi/info.json [hunk 3]
```diff
@@ -0,0 +1,76 @@
+{
+    "keyboard_name": "KG60 ANSI",
+    "url": "https://www.keygem.store/",
+    "maintainer": "Keygem",
```
Comment (fauxpark) [inline keyboards/keygem/kg60ansi/info.json:4]:
```suggestion
    "keyboard_name": "KG60 ANSI",
    "manufacturer": "FJLabs",
    "url": "https://www.keygem.store/",
    "maintainer": "Keygem",
    "usb": {
        "vid": "0x4B47",
        "pid": "0x0001",
        "device_version": "0.0.1"
    },
```

File: keyboards/keygem/kg60ansi/config.h [hunk 4]
```diff
@@ -0,0 +1,119 @@
...
+
+#include "config_common.h"
+
+/* USB Device descriptor parameter */
+#define VENDOR_ID    0x4B47 // FJLabs
+#define PRODUCT_ID   0x0001 // tf60ansi
+#define DEVICE_VER   0x0001 // Version 1
+#define MANUFACTURER    FJLabs
+#define PRODUCT         KG60ANSI
+
```
Comment (fauxpark) [inline keyboards/keygem/kg60ansi/config.h:28]:
```suggestion
```

File: keyboards/keygem/kg65rgbv2/info.json [hunk 5]
```diff
@@ -0,0 +1,83 @@
+{
+    "keyboard_name": "KG65 RGB V2",
+    "url": "https://www.keygem.store/",
+    "maintainer": "Keygem",
```
Comment (fauxpark) [inline keyboards/keygem/kg65rgbv2/info.json:4]:
```suggestion
    "keyboard_name": "KG65 RGB V2",
    "manufacturer": "FJLabs",
    "url": "https://www.keygem.store/",
    "maintainer": "Keygem",
    "usb": {
        "vid": "0x4B47",
        "pid": "0x0002",
        "device_version": "0.0.1"
    },
```

File: keyboards/keygem/kg65rgbv2/config.h [hunk 6]
```diff
@@ -0,0 +1,119 @@
...
+
+#include "config_common.h"
+
+/* USB Device descriptor parameter */
+#define VENDOR_ID    0x4B47 // FJLabs
+#define PRODUCT_ID   0x0002 // tf60v2
+#define DEVICE_VER   0x0001 // Version 1
+#define MANUFACTURER    FJLabs
+#define PRODUCT         KG65RGBv2
+
```
Comment (fauxpark) [inline keyboards/keygem/kg65rgbv2/config.h:28]:
```suggestion
```

**Review by fauxpark — CHANGES_REQUESTED** [review]

**Review by fauxpark — APPROVED** [review]

**Pull request merged**